_posts/en/newsletters/2018-10-30-newsletter.md

```diff
@@ -205,7 +205,8 @@ it to write a good description, and I doubt non-LN devs care -->{% endcomment %}
 - [LND #1981][] ensures that LND doesn't leak information about any of
   its peers that aren't advertising themselves as public nodes.
 
-- LND [#1535][LND #1535] and [#1512][LND #1512] adds the server-side
+- {:#lnd-1535-1512}
```

Comment:
This is still moving this one bullet point into its own `<ul>`. Can you move this Block Attribute List to after the text?

Reply:
actually, I've just done this myself and pushed

Reply:
I think that edit is fine, but are you sure you were testing the updated branch?  On my branch (ede049d564559e65471b41c0027d7a1b20c939c9, without your change), I don't see an extraneous `<ul>`:

```html
  </li>
  <li id="lnd-1535-1512">
    <p>LND <a href="https://github.com/lightningnetwork/lnd/issues/1535">#1535</a> and <a href="https://github.com/lightningnetwork/lnd/issues/1512">#1512</a> adds the server-side
```

Reply:
Yeah, looking at the GitHub forced push, it looks like you merged the old version of this with your one change.